doc/howto/cluster_recover.md

```diff
@@ -1,118 +1,121 @@
 (cluster-recover)=
 # How to recover a cluster
 
-You can tweak the amount of seconds after which a non-responding node will be
-considered offline by running:
+It might happen that one or several members of your cluster go offline or become unreachable.
+In that case, no operations are possible on this member, and neither are operations that require a state change across all members.
+See {ref}`clustering-offline-members` for more information.
 
-```bash
-lxc config set cluster.offline_threshold <n seconds>
+If you can bring the offline cluster members back or delete them from the cluster, operation resumes as normal.
+If this is not possible, there are a few ways to recover the cluster, depending on the scenario that caused the failure.
+See the following sections for details.
+
+```{note}
+When your cluster is in a state that needs recovery, most `lxc` commands do not work, because the LXD client cannot connect to the LXD daemon.
+
+Therefore, the commands to recover the cluster are provided directly by the LXD daemon (`lxd`).
+Run `lxd cluster --help` for an overview of all available commands.
 ```
 
 ## Recover from quorum loss
 
-Every LXD cluster has up to 3 members that serve as database nodes. If you
-permanently lose a majority of the cluster members that are serving as database
-nodes (for example you have a 3-member cluster and you lose 2 members), the
-cluster will become unavailable. However, if at least one database node has
-survived, you will be able to recover the cluster.
+Every LXD cluster has a specific number of members (configured through [`cluster.max_voters`](server)) that serve as voting members of the distributed database.
+If you permanently lose a majority of these cluster members (for example, you have a three-member cluster and you lose two members), the cluster loses quorum and becomes unavailable.
+However, if at least one database member survives, it is possible to recover the cluster.
 
-In order to check which cluster members are configured as database nodes, log on
-any survived member of your cluster and run the command:
+To do so, complete the following steps:
 
-```
-lxd cluster list-database
-```
+1. Log on to any surviving member of your cluster and run the following command:
 
-This will work even if the LXD daemon is not running.
+       sudo lxd cluster list-database
 
-Among the listed members, pick the one that has survived and log into it (if it
-differs from the one you have run the command on).
+   This command shows which cluster members are configured as database nodes.
+1. Pick one of the listed database nodes that is still online and log on to the machine (if it differs from the one you are already logged on to).
```

Comment:
@MathieuBordere when recovering from quorum loss manually, is it worth recommending that all LXD members be shutdown first before recovering the selected one as the new leader?

Reply:
Yes, I think it's worth recommending.